<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1342</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>2</width>
    <height>2</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Queue</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">gridline-color: rgb(21, 32, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="horizontalSlider_D1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_D2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Start_D1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Stop_D1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Стоп</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_properties">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>140</y>
      <width>284</width>
      <height>483</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>999</width>
      <height>999</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;hgfn&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;
gridline-color: rgb(255, 0, 0);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>84</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>55</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>42</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>Количество 
заявок в 
системе</string>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic UI Semibold</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Количество 
обработан-
ных заявок</string>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic UI Semibold</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Общее 
количество 
заявок</string>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic UI Semibold</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="9" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="9" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="9" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tableWidget_queue">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>618</width>
      <height>398</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
gridline-color: rgb(255, 0, 0);</string>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <property name="columnCount">
     <number>10</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>59</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>55</number>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QPushButton" name="pushButton_Start_D2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>560</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Stop_D2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>560</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Стоп</string>
    </property>
   </widget>
   <widget class="QLabel" name="simple_label_D1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Yu Gothic UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>D1</string>
    </property>
   </widget>
   <widget class="QLabel" name="simple_label_D2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Yu Gothic UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>D2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>1341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Bahnschrift SemiLight SemiConde&quot;;</string>
    </property>
    <property name="text">
     <string>	*Дисциплина обслуживания с абсолютными приоритетами*</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>150</y>
      <width>20</width>
      <height>472</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>16</width>
      <height>471</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_for_slader_D1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 15pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>100 мс</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_for_slader_D2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 15pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>100 мс</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
